Formula/siril.rb

```diff
@@ -0,0 +1,69 @@
+class Siril < Formula
+  desc "Astronomical image (pre-)processing application"
+  homepage "https://free-astro.org/index.php/Siril"
+  url "https://free-astro.org/download/siril-0.9.8.tar.bz2"
+  sha256 "ecb5477937afc02cc89cb07f4a7b99d2d0ab4cc5e715ec536e9be5c92a187170"
+  head "https://free-astro.org/svn/siril/"
+
+  bottle do
```

Comment:
Remove the bottle block

Reply:
Documentation says:
If there’s a bottle do block in the formula, don’t remove or change it; we’ll update it when we pull your PR
But ok. I'll remove the block